docs/en/metrics/metrics-installation.asciidoc

```diff
@@ -1,100 +1,107 @@
+(version 6.5 or later) for visualizing and managing it.
 
-NOTE: If your data uses nonstandard fields, you may need to modify some default <<configure-metrics-source,configuration settings>>.
+To ingest data, you can use {metricbeat} installed on each server you want to monitor, or
+third-party collectors that are configured to ship ESC-compliant data. <<metrics-fields-reference>>
+provides a list of all fields used in the {metrics-app}. 
 
-[float]
-=== Hosted Elasticsearch Service
-
-Skip installing and managing your own {es} and {kib} instance by using our hosted {es} Service.
-{ess-trial}[Try out the {es} Service for free].
+If your data uses nonstandard fields, you may need to modify some default <<configure-metrics-source,configuration settings>>.
+To modify configurations, use the <<configure-metrics-source,Settings tab>> in the {metrics-app}.
+Alternatively, see {kibana-ref}/infrastructure-ui-settings-kb.html[{metrics} settings] for
+a complete list of metrics configuration settings.
 
 [float]
-=== Install the stack yourself
-
-If you'd rather install the stack yourself,
-first see the https://www.elastic.co/support/matrix[Elastic Support Matrix] for information about supported operating systems and product compatibility.
-
-* <<install-elasticsearch-metrics>>
-* <<install-kibana-metrics>> (version 6.5 or later) with a basic license
-* <<install-beats-for-metrics>> (version 6.5 or later) on each of the systems you want to monitor
-
-[[install-elasticsearch-metrics]]
-=== Step 1: Install Elasticsearch
-
-Install an {es} cluster, start it up, and make sure it's running.
-
-. Verify that your system meets the
-https://www.elastic.co/support/matrix#matrix_jvm[minimum JVM requirements] for {es}.
-. {stack-gs}/get-started-elastic-stack.html#install-elasticsearch[Install Elasticsearch].
-. {stack-gs}/get-started-elastic-stack.html#_make_sure_elasticsearch_is_up_and_running[Make sure elasticsearch is up and running].
+[[before-you-begin-metricbeat]]
+=== Before you begin
 
-[[install-kibana-metrics]]
-=== Step 2: Install Kibana
-
-Install {kib}, start it up, and open up the web interface:
-
-. {stack-gs}/get-started-elastic-stack.html#install-kibana[Install Kibana].
-. {stack-gs}/get-started-elastic-stack.html#_launch_the_kibana_web_interface[Launch the Kibana Web Interface].
-
-[[install-beats-for-metrics]]
-=== Step 3: Set up and run {metricbeat}
-
-To start collecting metrics data, you need to install {metricbeat} and configure the {metricbeat} modules directly from {kib}.
-
-Alternatively, you can install {metricbeat} and configure the {metricbeat} modules yourself.
-
-[float]
-==== Install {metricbeat} from {kib}
+To get started quickly, you can spin up a deployment of our
+https://www.elastic.co/cloud/elasticsearch-service[hosted {ess}]. The deployment includes
+{es} and {kib}, and is available on AWS, GCP, and Azure.
+https://www.elastic.co/cloud/elasticsearch-service/signup?baymax=docs-body&elektra=docs[Try the {es}
+Service for free].
```

Comment:
All of these are available as [shared attributes](https://github.com/elastic/docs/blob/master/shared/attributes.asciidoc)

```suggestion
{ess-product}[hosted {ess}]. The deployment includes
{es} and {kib}, and is available on AWS, GCP, and Azure.
{ess-trial}[Try {ess} for free].
```